_layouts/tags.html

```diff
@@ -0,0 +1,35 @@
+<!-- Tag Layout Start -->
+
+<!doctype html>
+<html lang="{{ site.lang }}">
+
+  {% include head.html %}
+
+  <body>
+    {% include navigation.html %}
+
+    <section class="container content-section text-center">
+      <div class="row">
+        <div class="col-md-10 col-md-offset-1 col-xs-10 col-xs-offset-1">
+          {% assign tag = page.title %}
+
+          <h3 id="#tags">#Tags</h3>
+          <ul style="list-style-type: none;display: flex;flex-direction: column;align-items: center;padding: 0">
+            {% assign sorted_tags = site.tags | sort %}
+            {% for tag in sorted_tags %}
+              <li>
+                <a href="{{ site.baseurl }}/tags/{{ tag[0] }}.html">{{ tag[0] }}</a> ({{ tag[1].size }})
```

Comment:
Instead of just showing the counter next to the tag, let's make it more descriptive by saying `({{tag[1].size}} posts)`
![Screenshot 2024-10-22 at 10 38 49](https://github.com/user-attachments/assets/a8168c88-6926-46de-ad53-824b1c630256)

Reply:
Way more understandable, indeed!